### PR TITLE
refactored server code

### PR DESCRIPTION
**Pull request by ashishchoudhary001**
refactored server code and removed portal fcb

### CODE REVIEW

File: src/client/extension.ts [hunk 1]
```diff
@@ -23,70 +23,63 @@ export async function activate(
 
     _context = context;
 
-    const isPaportalFeatureEnabled = vscode.workspace
-        .getConfiguration("powerplatform-vscode")
-        .get("enablePortalFeatures");
-    if (isPaportalFeatureEnabled) {
```
[the remark this answers is not in the document]
Comment (ashishchoudhary001) [reply src/client/extension.ts]: Done

File: src/server/lib/PortalAutoCompleteHelper.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import {
```
[the remark this answers is not in the document]
Comment (ashishchoudhary001) [reply src/server/lib/PortalAutoCompleteHelper.ts]: Removed this file. Replaced this with two other files and both have the copyright header.

File: src/server/lib/PortalAutoCompleteHelper.ts [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    WorkspaceFolder
+} from 'vscode-languageserver/node';
+import { URL } from 'url';
+import * as path from 'path';
+import * as fs from 'fs';
+import * as YAML from 'yaml';
+
```
[the remark this answers is not in the document]
Comment (ashishchoudhary001) [reply src/server/lib/PortalAutoCompleteHelper.ts]: Removed this file and have split the functionality into two separate files. Have tried to stay away from using 'Helper', 'Utility' :)

**Review by davidjenni — APPROVED** [review]
thank you for the refactoring, looks waaaay better :-)

**Pull request merged**